### PR TITLE
Fix #2179: Add status in contact messages

### LINKED ISSUE #2179: Add status in contact messages
## Current Behaviour

If there are multiple EvalAI admins then it is not clear in contact messages if one admin has resolved it or not.

## Expected Behaviour
Add a choice field in the contact model with status as `pending` and `resolved` having default as `pending`.

## Deliverables
- [ ] Add the fields in the model.
- [ ] Display it in Contact admin.

### PR DESCRIPTION
**Pull request by Sanji515**
Deliverables

- [x] Added the status field in the Contact model.
- [x] Added status field in `apps/web/admin.py` in order to display in Contact Admin.

Screenshot (192)

### CODE REVIEW

**Comment by Sanji515** [comment]
@RishabhJain2018 @deshraj  Please review.

**Comment by kurianbenoy** [comment]
LGTM :)

File: apps/web/migrations/0006_added_status_field.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```
Comment (RishabhJain2018) [inline apps/web/migrations/0006_added_status_field.py]: Please rename the migrations to `0006_add_status_field_in_contact_model.py`

Comment (Sanji515) [reply apps/web/migrations/0006_added_status_field.py]: @RishabhJain2018 done.

**Comment by RishabhJain2018** [comment]
LGTM, Thanks for the fix @Sanji515 .

**Pull request merged**